website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -0,0 +1,683 @@
+may need to be performed to better understand how to gather this information.
+
+
+### Back Population of the ORCA Catalog
+
+Generally, the first two options listed will handle most use cases for adding
+information to the ORCA archive. However, for users that already have ORCA
+installations that are older than or equal to v3.x, a new workflow will need to
+be created to back populate the ORCA metadata catalog. Ideally, this workflow
+would be kicked off as part of the ORCA installation and/or migration.
+
+As a general breakdown of the flow, the Lambda or Worflow will need to do the
+following:
+
+1. Query Cumulus looking for Collections configured for ORCA.
+2. Using the list of Collections configured for ORCA, retrieve the granule and file information.
+3. Get a list of the files in each ORCA bucket.
+4. Match attributes from the ORCA file list like file name, key path, and size to file(s) in the Cumulus list.
+5. Create a message body from the information similar to what `copy_to_granule` will perform.
+6. Send the message body to the SQS FIFO queue so the database lambda can write the record to the database.
+
+There are likely several ways to optimize this flow. The above is just a general
+concept to help get the information needed to match files. Collection regex information
+could also possibly be used.
+
+Note that this concept of back populating by matching data is not 100% accurate
+and is a best attempt to pull data and match it without "re-ingesting" items
+back into ORCA. This approach will likely need lots of logging and reporting to
+let end users know where discrepancies and best guesses or misses occur. We
+should also talk with end users about this option prior to implementing to
+determine if this is needed.
+
+
+### Potential ORCA Catalog Population Architecture
+
+The following is a potential architecture for populating the ORCA catalog under
+normal use cases. Note that the SQS queue and `orca_catalog_ingest` lambda could
+be replaced with a GraphQL server if deemed appropriate.
+
+<MyImage
+    imageSource={useBaseUrl('img/Research-Reconciliation-Catalog-Population.png')}
+    imageAlt="System Context"
+    zoomInPic={useBaseUrl('img/zoom-in.svg')}
+    zoomOutPic={useBaseUrl('img/zoom-out.svg')}
+    resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
+/>
+
+
+## Possible Technologies for Enabling the Work
+
+The following is a short list of technologies that could be leveraged during this
+work. These technologies fall within current architectures for Cumulus and
+extensions and may contribute to more rapid development of features as we progress.
+A full list of AWS services available in NGAP is available [here](https://wiki.earthdata.nasa.gov/display/ESKB/AWS+Services+Approval+Status+Page).
+
+
+### API Gateway
+
+The ORCA API Gateway provides an entry point for the Cumulus dashboard and users
+to interact directly with ORCA. For more information on the ORCA API gateway see
```

Comment:
to interact directly with ORCA. For more information on the ORCA API gateway, see